soh/soh/Enhancements/debugger/debugSaveEditor.h

```diff
@@ -522,6 +522,8 @@ const std::vector<FlagTable> flagTables = {
         { RAND_INF_HAS_OCARINA_C_DOWN, "RAND_INF_HAS_OCARINA_C_DOWN" },
         { RAND_INF_HAS_OCARINA_C_LEFT, "RAND_INF_HAS_OCARINA_C_LEFT"},
         { RAND_INF_HAS_OCARINA_C_RIGHT, "RAND_INF_HAS_OCARINA_C_RIGHT"},
+        
+        { RAND_INF_HAS_WALLET, "RAND_INF_HAS_WALLET" },
```

Comment:
Not sure why this space is here, not a huge deal but if you're going to be changing anything else may as well remove it.

Reply:
The space is intentional to separate it from the ocarina button lines.

Reply:
Oh ok, I guess that's how the original authors chose to format that list...